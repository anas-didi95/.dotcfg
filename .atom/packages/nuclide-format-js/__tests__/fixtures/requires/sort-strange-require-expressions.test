const a = require('a')()()().a().b.c[0];
require('b')().y;
require('b')().x;
const D = require('meh').a.b.c('123');
const C = 'not'.a.require('meh');
const E = require('TableWidthsConfig').getWidthConfig().CAMP_GROUP_PANE;
a(D, E);
