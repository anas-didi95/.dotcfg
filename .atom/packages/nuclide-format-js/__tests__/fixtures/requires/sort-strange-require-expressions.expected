require('b')().x;
require('b')().y;

const D = require('meh').a.b.c('123');
const E = require('TableWidthsConfig').getWidthConfig().CAMP_GROUP_PANE;

const a = require('a')()()().a().b.c[0];

const C = 'not'.a.require('meh');
a(D, E);
